lib/settings.coffee

```diff
@@ -1,8 +1,15 @@
 module.exports =
   config:
+
+    fontSize:
+      title: 'Font Size'
+      description: 'Change the UI font size. Needs to be between 10 and 20.'
+      type: 'string'
```

Comment:
I'm not at the computer right now. But you should be able to give it a list
of types integer and null. Check out the config API docs

On Wednesday, May 27, 2015, Lee Dohm <notifications@github.com> wrote:

> In lib/settings.coffee
> <https://github.com/atom/one-dark-ui/pull/73#discussion_r31199469>:
>
> > @@ -1,8 +1,15 @@
> >  module.exports =
> >    config:
> > +
> > +    fontSize:
> > +      title: 'Font Size'
> > +      description: 'Change the UI font size. Needs to be between 10 and 20.'
> > +      type: 'string'
>
> I'm pretty certain that if you don't specify a default, the default will
> be undefined.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/one-dark-ui/pull/73/files#r31199469>.
>